### ISSUE TITLE
Versions with underscore seperators not detected correctly

### ISSUE THREAD

**Issue by cajus** [flathub-infra/flatpak-external-data-checker#448]
I'd like to check for updates of the openldap package, and they've sadly used underscore instead of dot seperators for the version. Testing the update using this snippet

```yml
id: org.openldap.check
modules:
- name: openldap
  config-opts:
    - --disable-static
    - --disable-slapd
    - --disable-slurpd
    - --disable-bdb
    - --disable-hdb
    - --with-tls=openssl
  cleanup:
    - /bin
  sources:
    - type: archive
      url: "https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.9.tgz"
      sha256: 2cb7dc73e9c8340dff0d99357fbaa578abf30cc6619f0521972c555681e6b2ff
      x-checker-data:
        type: anitya
        stable-only: true
        project-id: 2551
        url-template: "https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-${major}.${minor}.${patch}.tgz"
```

runs into

```
$ flatpak run org.flathub.flatpak-external-data-checker --check-outdated test.yml 
INFO    src.manifest: Checking 1 external data items
INFO    src.manifest: Started check [1/1] archive openldap/openldap-2.6.9.tgz (from test.yml)
ERROR   src.manifest: Failed to check archive openldap/openldap-2.6.9.tgz with AnityaChecker: Error downloading upstream source: 404, message='Not Found', url='https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2._.6.tgz'
WARNING src.main: Check finished with 1 error(s)
```

. Any idea how I can work around that?

**Comment by bbhtt** [comment]
Seems to work fine here, I don't see where it is using underscore here https://www.openldap.org/software/download/OpenLDAP/openldap-release/.

You can also use

```yaml
url-template: "https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-$version.tgz"
```
